src/style.css

```diff
@@ -1,3 +1,108 @@
-body {
+html {
+  --color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  text-transform: uppercase;
+}
+
+.logo {
+  margin-left: 50px;
+  display: flex;
+  align-items: center;
+}
+
+header .logo img {
+  height: 40px;
+}
+
+nav {
+  flex-grow: 1;
+  display: flex;
+  justify-content: flex-end;
+  margin-left: 419px;
+  margin-right: 50px;
+  height: 60px;
+}
+
+ul {
+  display: flex;
+  align-items: center;
+  list-style: none;
```

Comment:
Don't use tag names for styling. Instead, use classes to style your elements. For example, apply a class to the `ul` element and style that class.